src/lib/utils/typedObject.ts

```diff
@@ -0,0 +1,5 @@
+// https://stackoverflow.com/a/65117465/3185307
```

Comment:
Could you add a more descriptive comment than just the link ? It would help to understand the purpose of this snippet without having to open the link.

Thinking about something like:

```js
/*
 * Typed `Object.keys` to return `Array<keyof TObject>` instead of `Array<string>`
 * source: https://stackoverflow.com/a/65117465/3185307
 */
```